lti_consumer/models.py

```diff
@@ -64,6 +68,29 @@ class LtiConfiguration(models.Model):
         blank=True,
     )
 
+    # LTI 1.3 Related variables
+    _lti_1p3_platform_private_key = models.TextField(
+        blank=True,
+        help_text="Platform's generated Private key. Keep this value secret.",
+    )
+
+    _lti_1p3_platform_private_key_id = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="Platform's generated Private key ID",
+    )
+
+    _lti_1p3_platform_public_jwk = models.TextField(
+        blank=True,
+        help_text="Platform's generated JWK keyset.",
+    )
+
+    lti_1p3_client_id = models.CharField(
+        max_length=255,
+        default=str(uuid.uuid4()),
```

Comment:
I think you might want to use [`UUIDField`](https://docs.djangoproject.com/en/3.1/ref/models/fields/#uuidfield) instead here

Reply:
@pcockwell I want to use a `string` here, like on the old implementation. I'm only using `uuid4` since it's an easy way to generate a random string.